docs/docsite/rst/intro_windows.rst

```diff
@@ -178,6 +178,15 @@ To see what tickets if any you have acquired, use the command klist
 
    klist
 
+Automatic kerberos ticket management
+------------------------------------
+
+Ansible defaults to automatically managing kerberos tickets (as of Ansible 2.3) when both username and password are specified for a host that's configured for kerberos. A new ticket is created in a temporary credential cache for each host, before each task executes (to minimize the chance of ticket expiration). The temporary credential caches are deleted after each task, and will not interfere with the default credential cache.
```

Comment:
@nitzmahone I have to agree with @dharmabumstead here. Until we have versioned docs, this is good info to lead with. As the usefulness declines, we should go back and revise/edit as needed instead of writing for a future point in time. In my experience, I've found that customers skim docs more than read them, and things in "( )" are often skimmed right over/ignored.